beta-src/src/utils/state/gameApiSlice/reducers/processMapClick.ts

```diff
@@ -22,14 +26,63 @@ import startNewOrder from "../../startNewOrder";
 import updateOrder from "../../updateOrder";
 import updateOrdersMeta from "../../updateOrdersMeta";
 
-function canUnitMove(orderMeta: OrderMeta, territory: Territory): boolean {
+function canUnitMoveTo(orderMeta: OrderMeta, territory: Territory): boolean {
   const { allowedBorderCrossings } = orderMeta;
-  console.log({ allowedBorderCrossings, territory });
   return !!allowedBorderCrossings?.find(
     (border) => TerritoryMap[border.name].territory === territory,
   );
 }
 
+function canUnitMoveToRegion(
+  orderMeta: OrderMeta,
+  territory: Territory,
+): boolean {
+  const { allowedBorderCrossings } = orderMeta;
+  return !!allowedBorderCrossings?.find(
+    (border) =>
+      TerritoryMap[border.name].territory === territory ||
+      TerritoryMap[border.name].parent === territory,
+  );
+}
+
+function canSupporteeMoveToOrHoldAtRegion(
```

Comment:
Can't you use the existing `order.supportHoldChoices` and `order.supportMoveChoices` for this?

I was imagining we would modify `supportMoveChoices` to be Territory -> Territory[] instead of Territory[] -> Territory, and then as long as those classes were written correctly all this logic should be simple.

Reply:
In fact, we should combine this to just `order.supportChoices: Territory -> Territory[]` and then the logic will simplify even further.

Reply:
I didn't think you could use order.supportHoldChoices and order.supportMoveChoices for this right now because the supportee may be another power, and we aren't computing orders or OrdersMeta for other powers.

Reply:
Yeah, I think `supportHoldChoices` and `supportMoveChoices` is supposed to be a list of valid support hodls and support moves *for this unit*.

Not sure if it's bug-free, but since we're already computing these things it would be ideal if we could use them.